decidim-core/lib/decidim/core.rb

```diff
@@ -292,6 +292,13 @@ def self.seed!
     ";"
   end
 
+  # Exposes a configuration option: HTTP_X_FORWADED_HOST header follow-up.
```

Comment:
Is it really necessary to have this as a setting or we could assume that we don't need this header for anything on the application level? I mean, I can't think on any case where Decidim could need this, AFAIK this header is only relevant for logging the real host when there's a reverse proxy.  

More information: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/X-Forwarded-Host

Reply:
I don't like choosing in place of an administrator. But if this is not needed, I will remove it. Would you want me to remove it?

Reply:
Lets wait for @decidim/core opinion first as there could be cases that I'm not taking into account...

Reply:
There are many Decidim deploys that live behind proxies and reverse proxies, of many types, that send this header. I would keep the configuration option just in case and add a deprecation warning.

Reply:
Ok, so let's keep the setting here